src/times.js

```diff
@@ -0,0 +1,12 @@
+const randomBool = () => Math.random() > 0.5;
```

Comment:
@AlekseyGoremykin don't mix tasks. this branch should not contain `src/times.js`